Documentation/installation/k8s-install-talos-linux.rst

```diff
@@ -59,6 +59,17 @@ Next, generate the configuration files for the Talos cluster by using the
 
 **Install Cilium**
 
+.. warning::
+
+    When using tunneling mode and enabling `Kube-Proxy replacement<kubeproxy-free>`
+    together with Talos' `Forwarding kube-dns to Host DNS`_ (enabled by default
+    since Talos 1.8+), you need to set ``bpf.hostLegacyRouting`` to ``true`` as
```

Comment:
Based on code behaviour and past reports, I'd expect that `native` routing mode is affected in the same way, and also doesn't work when using BPF Host Routing.

So it might be easiest to just express this as general limitation, even though it accidentally worked on older releases when `overlay` routing mode was configured.